caffe2/core/hip/context_hip.h

```diff
@@ -175,7 +175,7 @@ class HIPContext final : public BaseContext {
     return hip_objects_.GetStream(gpu_id, stream_id);
   }
 
-  rocblas_handle rocblas_handle() {
+  rocblas_handle rocblashandle() {
```

Comment:
looks like this fix is unrelated to docker image? since the docker PR will takes longer to land (need to rebuild docker images and do testing and update build config) could you put this fix in a separate PR so we can land it sooner?

Reply:
It's somewhat related as that the docker image uses `gcc` as the base compiler which actually complains about this. `llvm` tolerates it. I can make a separate PR for it though.

Reply:
@iotamudelta ping to split out this into separate PR

Reply:
@bddppq The PR is in preparation. Should come through CI hopefully soon.

Reply:
@iotamudelta could you remove the rocblas_handle changes from this PR?